From 7da43aed4e7fd055995226a4433c79c324b5ff2a Mon Sep 17 00:00:00 2001
From: Beleswar Padhi <b-padhi@ti.com>
Date: Tue, 18 Mar 2025 15:50:21 +0530
Subject: [PATCH 5/5] arm64: defconfig: Enable TI K3 M4 Remoteproc driver

Enable the TI_K3_M4_REMOTEPROC driver to enable IPC on M55 remote
processor on tda54-vdk platform.

Signed-off-by: Beleswar Padhi <b-padhi@ti.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index ab46ae6f9bca..5767fb443ae6 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -1801,3 +1801,4 @@ CONFIG_CORESIGHT_CTI=m
 CONFIG_MEMTEST=y
 CONFIG_SAMPLES=y
 CONFIG_SAMPLE_TDA54_MAILBOX=m
+CONFIG_TI_K3_M4_REMOTEPROC=m
-- 
2.34.1

